### DISCUSSION TITLE
How to do undo "brew rubocop"

### DISCUSSION THREAD

**Discussion by isadon** [Homebrew/discussions#3186] [Everyday usage]
Brew rubocop turns developer mode on and installs a bunch of gems. How do I undo whatever `brew rubocop` just did?

**Comment by SMillerDev** [comment]
The gems won't affect brew usage. But you can run `brew developer off` to turn off developer mode.

**Reply by isadon** [reply]
I don't want the gems that it installed taking up disk space though. How do I remove those?
